README.md

```diff
@@ -1,18 +1,25 @@
 This is the readme for the Python for Win32 (pywin32) extensions source code.
 
-See CHANGES.txt for recent changes.
+See [CHANGES.md] for recent changes.
```

Comment:
Link broken.

Reply:
Looks like he missed to commit his changes on CHANGES.md :confused: